src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -30,6 +30,15 @@
     }
 }
 
+.ez-side-menu,
+.ez-context-menu {
+    .ez-icon {
+        width: 1.85rem;
```

Comment:
I think 1.85rem is 29.6px. Did you mean 1.875rem?

Reply:
Updated. Thanks! 👍